backend/src/langchain.ts

```diff
@@ -50,9 +50,11 @@ function initQAModel(level: LEVEL_NAMES, Prompt: string) {
 		'QA prompt template'
 	);
 	console.debug(`QA chain initialised with model: ${modelName}`);
-	return RetrievalQAChain.fromLLM(model, documentVectors.asRetriever(), {
-		prompt: promptTemplate,
-	});
+	return RetrievalQAChain.fromLLM(
+		model,
+		documentVectors.asRetriever(documentVectors.memoryVectors.length),
```

Comment:
This is the actual fix! Use all docs from the vector store in our retrieval context.